index.js

```diff
@@ -61,6 +62,21 @@ async function installRelativeDeps() {
   }
 }
 
+async function watchRelativeDeps() {
+  const projectPkgJson = readPkgUp.sync()
+
+  const relativeDependencies = projectPkgJson.package.relativeDependencies
+
+  if (!relativeDependencies) {
+    console.warn("[relative-deps][WARN] No 'relativeDependencies' specified in package.json")
+    process.exit(0)
+  }
+
+  Object.values(relativeDependencies).forEach(path => {
+    fs.watch(path, { recursive: true }, debounce(installRelativeDeps, 500))
```

Comment:
It would be nice here to only reinstall the specific dependency that triggered the change rather than all. But since that is usually only one, I think this is fine for now and could be optimized later.